botocore/utils.py

```diff
@@ -3315,3 +3315,12 @@ def _serialize_if_needed(self, value, iso=False):
                 return value.isoformat()
             return value.strftime('%Y-%m-%dT%H:%M:%S%Z')
         return value
+
+
+def get_crt_version():
+    try:
+        import awscrt
+
+        return awscrt.__version__
+    except AttributeError:
+        return None
```

Comment:
Ideally this would remain private but I don't know if we can do that cleanly while reusing. Perhaps we at least leave a note indicating it's not for public use and subject to change.

Reply:
I found a way to keep _get_crt_version private 66ad86a, at the cost of one cross-module call to a private function.